script/Common/00_02.txt

```diff
@@ -321,32 +321,32 @@ Red.
 [sha:c66e864fe1584af9a6166daf27efa4b8278a1481]{
 -- Page 14, Offset 72.
 -- トマトみたいに　あかい水。
-Red water. Red like a tomato.
+Red liquid, the color of a tomato.
```

Comment:
I guess when I wrote this I was thinking (トマトみたいにあかい)水, because it makes sense for tomatoes to apply red and less so wet. If we instead treat this as トマトみたいに(あかい水), I might edit this a bit to be
> Red and wet, like tomato juice.

because an intact tomato is usually quite dry.

Reply:
I think the space is there to signal トマトみたいに(あかい水). That makes sense to mean, at least.
I like your suggestion though.